CheckValues/Results.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Text;
+using WaterLog_Backend;
+using WebApplication1;
+using WaterLog_Backend.Models;
+using WaterLog_Backend.Controllers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckValues
+{
+    public class Results
+    {
+
+        public MonitorsController getController()
+        {
+            var builder = new ConfigurationBuilder();
+            builder.AddUserSecrets<Startup>();
+            var config = builder.Build();
+            //string mySecret = config["Localhost:ConnectionString"];
```

Comment:
GC, please remove or add TODO: with an explanation so whoever does it later has enough context to understand.

Reply:
done